### ISSUE TITLE
Failed installation because of wrong metadata in python3-indy package

### ISSUE THREAD

**Issue by esune** [hyperledger/indy-sdk#2355]
Installation of `python3-indy` fails because of incorrect package metadata.
The package is being installed with versioning `python3-indy>=1.11.1<2` as per [Aries Cloudagent Python requirements](https://github.com/hyperledger/aries-cloudagent-python/blob/master/requirements.indy.txt).
Python version: `3.9.1`
Pip version: `20.3.3`
Output:
```
$ pip install -r requirements.indy.txt
Collecting python3-indy>=1.11.1<2
  Downloading python3-indy-1.16.0-rc-170.tar.gz (45 kB)
     |████████████████████████████████| 45 kB 3.4 MB/s 
ERROR: Requested python3-indy>=1.11.1<2 from https://files.pythonhosted.org/packages/28/7a/bc971de34abcbe07f3e9c10ca26d2aa61f6344a96b0fb0f92756d225cb0b/python3-indy-1.16.0-rc-170.tar.gz#sha256=05c69a8960430a50b22ff233c82cd27174c80077d6b6170b3c83409e5d8c4a4a (from -r requirements.indy.txt (line 1)) has different version in metadata: '1.16.0'
```

Current workaround: use `--use-deprecated=legacy-resolver` when installing dependencies, however this is not a long-term solution since pip 21 will stop allowing this.

**Comment by ianco** [comment]
This is due to a combination of (a) installing an RC release (the version has "-rc-170" appended to it), and (b) using the latest pip (older versions of pip didn't error out on this)

This should be fixed once the 1.16.0 release is officially released

**Comment by esune** [comment]
I am surprised the `pip` resolver even tries to fetch a `rc`, maybe a flag is missing in the pre-released package that would make `pip` skip it?
As long as we know the behaviour is there and it will just work with the new resolver correctly when things are finalized it's good though, I just wanted to raise the point early because of its concurrence with the new pip resolution strategy.
